### PR TITLE
chore(readme): add logo and link

### PR DESCRIPTION
**Pull request by AugustinMauroy**
Follow readme flavor of other repo

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,11 @@
-# OpenJS Foundation Collaborator Summit
+<a href="https://openjsf.org">
+   <picture>
+     <source media="(prefers-color-scheme: dark)" srcset="https://raw.githubusercontent.com/openjs-foundation/artwork/main/openjs_foundation/openjs_foundation-logo-horizontal-color-dark_background.svg" />
+     <img src="https://raw.githubusercontent.com/openjs-foundation/artwork/main/openjs_foundation/openjs_foundation-logo-horizontal-color.svg" width="200" alt="Openjs Foundation Logo" />
```
Comment (ljharb) [inline README.md:4]:
```suggestion
     <img src="https://raw.githubusercontent.com/openjs-foundation/artwork/main/openjs_foundation/openjs_foundation-logo-horizontal-color.svg" width="200" alt="OpenJS Foundation" />

```

**Review by ljharb — APPROVED** [review]

**Review by ruyadorno — APPROVED** [review]

**Comment by mcollina** [comment]
Isn't this repo going back to Node.js? If that's the case we don't need this PR

**Comment by AugustinMauroy** [comment]
Yes, discussions are underway. But I don't know if it's been successful.

So depending on the outcome of this discussion, this page will have to be updated with the node logo.

**Review by mcollina — APPROVED** [review]
lgtm

**Comment by chrisgervang** [comment]
Where can I learn about the repo move discussion? We've been using this for [openvisualization.org](https://www.openvisualization.org/), which isn't apart of Node.js but is apart of OpenJS.

**Comment by AugustinMauroy** [comment]
Hey @mcollina 👋
I think we can merge this pr.

**Pull request closed without merging**